File: playground/BOJ/boj_2743/b2743.c
// 단어 길이 재기
// https://www.acmicpc.net/problem/2743

#include <stdio.h>
#include <string.h>

int main() {
  char inputWord[101];
  int lengthOfWord;
  int result;

  scanf("%100s", inputWord); // 최대 100개의 문자를 읽도록 지정하여 버퍼 오버플로우 방지
  lengthOfWord = strlen(inputWord);

  result = lengthOfWord;

  printf("%d", result);

  return 0;
}

File: Online_Judge_Problems/BOJs/boj_2743/b2743.js
// 단어 길이 재기
// https://www.acmicpc.net/problem/2743

const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

let input = []

rl.on('line', (line) => {
  input.push(line)
})

rl.on('close', () => {
  const lengthOfWord = input[0].length
  const result = lengthOfWord

  console.log(result)
})